### PR TITLE
fix: list params swapped and next token set

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

**Review by dsharma-dc — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors try

File: tests/bdd/features/snapshot/csi/controller/operations.feature [hunk 1]
```diff
@@ -9,9 +9,13 @@ Feature: Snapshot - CSI Controller Operations
         Then the creation should succeed
 
     Scenario: Delete Snapshot Operation is implemented
+        Given a single replica volume
```
Comment (tiagolobocastro) [inline tests/bdd/features/snapshot/csi/controller/operations.feature]:
```suggestion
        Given we have a single replica volume
```

File: tests/bdd/features/snapshot/csi/controller/operations.feature [hunk 2]
```diff
@@ -9,9 +9,13 @@ Feature: Snapshot - CSI Controller Operations
...
+        And a snapshot is created for that volume
         When a DeleteSnapshotRequest request is sent to the CSI controller
         Then the deletion should succeed
 
     Scenario: List Snapshot Operation is implemented
+        Given a single replica volume
```
Comment (tiagolobocastro) [inline tests/bdd/features/snapshot/csi/controller/operations.feature]:
```suggestion
        Given we have a single replica volume
```

**Review by tiagolobocastro — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**